src/Checker/InvoiceCustomerRelationCheckerInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\InvoicingPlugin\Checker;
+
+use Sylius\Component\Core\Model\CustomerInterface;
+
+interface InvoiceCustomerRelationCheckerInterface
+{
+    public function check(string $invoiceId, CustomerInterface $customer): void;
```

Comment:
I'm missing the docblock about the exception thrown when a customer can't access the given invoice.